### ISSUE TITLE
Rename empirical and analytic null methods

### ISSUE THREAD

**Issue by tsalo** [neurostuff/NiMARE#421]
From https://github.com/neurostuff/NiMARE/issues/389#issue-740102784:

> Probably we should rename the options too, as "empirical" is a bit confusing, IMO. I would suggest calling them "approximate" and "montecarlo", which would intuitively convey to users both that (a) the "approximate" approach doesn't really give the 'correct' answer (because it's biased by the independence assumption), and (b) the "montecarlo" approach is based on resampling and is likely to take a long time. Then the docs could more explicitly lay out the tradeoff, taking pain to emphasize that the "approximate" approach is actually conservative, so if you're willing to accept reduced sensitivity, it makes sense to just always use that.

TL;DR: Rename "empirical" to "montecarlo" and "analytic" to "approximate".

**Comment by tsalo** [comment]
@tyarkoni what should we name the "old" empirical method? I know it will go largely undocumented but it will still need to be accessible and I'd like to have an informative name for it.

**Comment by tsalo** [comment]
Now that #424 has been merged, this is easy enough to do and I plan to tackle it soon.

Pinging @tyarkoni about the best terminology for the old empirical approach.
